src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  const shortTerm = 3;
+  const shorTermDiscount = 20;
+  const longterm = 7;
```

Comment:
The constant `longterm` should be in camelCase to follow JavaScript naming conventions. Please rename it to `longTerm`.